docs/web/index.md

```diff
@@ -577,6 +577,8 @@ Services that use the sda-spring-boot-commons:
 - are configurable by the property or environment variable `METADATA_FIELDS` to be aware of the
   metadata used in a specific environment
 
+### Async processes
```

Comment:
The new headline confuses me. We have an abstract here about the metadata context for services that are not interested in it. Actually if we need more sections with subheadlines it is imho too complicated to support it.